File: Задание 5/7.py
c = 0  
pn = None  
while True:  
    n = int(input("Введите натуральное число (0 для завершения): ")) 
    if n == 0:  
        break      
    if pn is not None and n > pn:
        c += 1  
    pn = n
print(f"Количество элементов, больших предыдущего: {c}")

File: 2 семестр /2 КТ /7.java
import java.util.*;
public class UniqueWords {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Введите строку слов:");
        String input = scanner.nextLine();
        String[] words = input.split("\\s+");
        LinkedHashSet<String> uniqueWords = new LinkedHashSet<>(Arrays.asList(words));
        System.out.println(String.join(" ", uniqueWords));
    }
}
